contrib/mod_tls.c

```diff
@@ -10674,13 +10677,28 @@ MODRET tls_prot(cmd_rec *cmd) {
     return PR_ERROR(cmd);
   }
 
+  c = find_config(main_server->conf, CONF_PARAM, "TLSOptions", FALSE);
+  while (c != NULL) {
+    unsigned long opts = 0;
+
+    pr_signals_handle();
+    opts = *((unsigned long *) c->argv[0]);
+    tls_opts |= opts;
+
+    c = find_config_next(c, c->next, CONF_PARAM, "TLSOptions", FALSE);
+  }
+
   if (!(tls_flags & TLS_SESS_PBSZ_OK)) {
-    pr_response_add_err(R_503,
-      _("You must issue the PBSZ command prior to PROT"));
+    if ( tls_opts & TLS_OPT_REQUIRE_PBSZ ) {
```

Comment:
Style nit: the ProFTPD coding style does not use spaces after an open parenthesis (`(`), or before the close parenthesis (`)`), thus:

    if (tls_opts & TLS_OPT_REQUIRE_PBSZ) {

would be more preferred.